src/pages/RepoPage/FlagsTab/constants.ts

```diff
@@ -13,6 +13,7 @@ export const MEASUREMENT_TIME_INTERVALS = {
 } as const
 
 export const AFTER_DATE_FORMAT_OPTIONS = {
+  ALL_TIME: { months: 6 },
```

Comment:
do we only have 6 months worth of data? or is this just wip

Reply:
There's some logic in useMeasurementVariables where if we select "All Time" it uses the oldestCommitAt timestamp, so that 6 month value shouldn't matter ideally. I had to add _some_ value though in the event oldestCommitAt doesn't exist (it's typed out as potentially undefined or null so there's a chance), so I wanted to fall back to the next "highest" value which in this case was 6 months.

This is due to us keying off AFTER_DATE_FORMAT_OPTIONS if oldestCommitAt doesn't exist, where before it likely would've returned undefined, now we're returning a legitimate value.

I can def add a comment up here for clarity if you think it'd be worthwhile